script/Ciel/Day 13/13_01_01_CIEL13_5B.txt

```diff
@@ -719,6 +721,7 @@ That wall... What's it about to do? What's it %{i}doing%{/i}?
 [sha:f854d7701f03873ce8f9a4e2a3488f7bfc6d4b05]{
 -- Page 610, Offset 35020.
 -- 「早く、早く俺を上にあげろ！　上げてくれよ！」
+// Maybe change this? Trying to get upsies over the wall
```

Comment:
what's the issue with this one?

Reply:
Reads fine to me too, but there was a thought that perhaps it wasn't clear. If it seems good to you too inclined to just leave it.

Reply:
it's fine to me.

Reply:
Was thinking something like this
>Come on, hurry up and lift me up already! Please, help me!